### PR TITLE
DE32856 - Adding Lit component that actually has the iframe in order …

### PR DESCRIPTION
**Pull request by pbrown-d2l**
…to be able to use the allow attribute in react 15

### CODE REVIEW

File: src/react-valence-ui-iframe.js [hunk 1]
```diff
@@ -11,72 +8,42 @@ var ResizingIframe = React.createClass({
...
-		};
+		return {};
 	},
 	componentDidMount: function() {
+		this.frame.addEventListener('d2l-iframe-wrapper-for-react-load', this.handleOnLoad);
+		this.frame.addEventListener('d2l-iframe-wrapper-for-react-resize', this.callbackWrapper);
 		this.updateProgress(0);
 	},
+	componentWillUnmount() {
+		this.frame.removeEventListener('d2l-iframe-wrapper-for-react-load', this.testLoad);
```
Comment (jstefaniuk-d2l) [inline src/react-valence-ui-iframe.js:19]:
```suggestion
		this.frame.removeEventListener('d2l-iframe-wrapper-for-react-load', this.handleOnLoad);
		this.frame.removeEventListener('d2l-iframe-wrapper-for-react-resize', this.callbackWrapper);
```

**Comment by ChrisKraljevicD2L** [comment]
I think some travis configs may need to change though https://travis-ci.com/github/Brightspace/react-valence-ui-iframe/builds/161851861#L843

**Comment by ChrisKraljevicD2L** [comment]
Actually yeah they definitely do: [it's currently using node 4](https://github.com/Brightspace/react-valence-ui-iframe/blob/master/.travis.yml#L3)

**Comment by pbrown-d2l** [comment]
hmm. I get the same failure locally on Master. I can't see if this has ever passed. It is looking for jasmine-core which is not referenced here

**Pull request merged**